src/main/java/org/gridsuite/shortcircuit/server/entities/ShortCircuitAnalysisResultEntity.java

```diff
@@ -43,9 +42,13 @@ and having a constant hashCode() causes performance issues.
     @Setter
     private Set<FaultResultEntity> faultResults;
 
-    public ShortCircuitAnalysisResultEntity(UUID resultUuid, Instant writeTimeStamp, Set<FaultResultEntity> faultResults) {
+    @Column(name = "debug_file_location")
+    private String debugFileLocation;
+
+    public ShortCircuitAnalysisResultEntity(UUID resultUuid, Instant writeTimeStamp, Set<FaultResultEntity> faultResults, String debugFileLocation) {
         this.resultUuid = resultUuid;
         this.writeTimeStamp = writeTimeStamp;
+        this.debugFileLocation = debugFileLocation;
```

Comment:
Maybe we'll have to evolve this model to be able to store few debug file locations. To be discussed with the PO's then